airssample_fr.xml

```diff
@@ -310,39 +310,39 @@ Dates : DATES [French]</Notes>
                     <Confidential>false</Confidential>
                 </AKA>
                 <AKA>
-                    <Name>FORMER_ORG 1 [French]</Name>
+                    <Name>ANCIENNE_ORG 1 [French]</Name>
                     <Confidential>false</Confidential>
-                    <Description>Former Name : 2001</Description>
+                    <Description>Nom antérieur : 2001</Description>
                 </AKA>
                 <AKA>
-                    <Name>FORMER_ORG 2 [French]</Name>
+                    <Name>ANCIENNE_ORG 2 [French]</Name>
                     <Confidential>false</Confidential>
-                    <Description>Former Name : 2008</Description>
+                    <Description>Nom antérieur : 2008</Description>
                 </AKA>
                 <AKA>
-                    <Name>FORMER_ORG 3 [French]</Name>
+                    <Name>ANCIENNE_ORG 3 [French]</Name>
                     <Confidential>false</Confidential>
-                    <Description>Former Name</Description>
+                    <Description>Nom antérieur</Description>
                 </AKA>
                 <AKA>
-                    <Name>LEGAL_ORG [French]</Name>
+                    <Name>JURIDIQUE_ORG [French]</Name>
                     <Confidential>false</Confidential>
                     <Description>Dénomination sociale</Description>
                 </AKA>
                 <Phone TollFree="false" Confidential="false">
                     <PhoneNumber>OFFICE_PHONE [French]</PhoneNumber>
-                    <Description>Office</Description>
-                    <Type>Voice</Type>
+                    <Description>Bureau</Description>
```

Comment:
Phone Descriptions and types are current fixed (don't change based on language) so they shouldn't be translated here.